pkg/sitewise/client/client.go

```diff
@@ -101,7 +102,7 @@ func GetClient(region string, settings AWSSiteWiseDataSourceSetting, provider aw
 		swcfg.Endpoint = aws.String(settings.Endpoint)
 	}
 
-	if settings.Region == "Edge" {
+	if settings.Region == models.EDGE_REGION {
```

Comment:
I didn't see the definition of `models.EDGE_REGION`. Is there any change not commited?

Reply:
it's within `pkg/models/setting.go` I believe